src/OrbitGl/App.cpp

```diff
@@ -2549,20 +2549,25 @@ Future<void> OrbitApp::AddDefaultFrameTrackOrLogError() {
     // from the list until one of them is loadable.
     if (preset.has_value() &&
         GetPresetLoadState(preset.value()).state == orbit_data_views::PresetLoadState::kLoadable) {
+      std::vector<std::filesystem::path> preset_module_paths = preset.value().GetModulePaths();
       orbit_base::ImmediateExecutor immediate_executor{};
-      return LoadPreset(preset.value())
```

Comment:
This is now missing the loading of the actual preset I think. We only load the modules below but we don't instrument the functions and set the frame track.

Reply:
Ohh, you are right. With Henning we understood that LoadPresetModule() was doing a different thing.

Reply:
sorry. I was right before. LoadPresetModule is loading a preset from a particular module. So the code was right.

What do you think about renaming LoadPresetModule to LoadPresetFromModule?

Reply:
Oh, right! Sorry for the noise! 

Yeah, I was mislead by the name and stopped reading through the code too early. I'm not sure if `LoadPresetFromModule` - I don't get this either. 
What does it actually do? Open a preset that requires a only exactly one module and return an error if that module does not exist? A comment in the header might help.

Reply:
Added a comment:
`// Load the functions and add frame tracks from a particular module of a preset file.`

Let me know what you think